pages/ksphere/konvoy/latest/release-notes/index.md

```diff
@@ -15,6 +15,48 @@ enterprise: false
 
 <p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>
 
+### Version v1.4.2 - Released 24 March 2020
+
+#### Disclaimer
+
+-   The default value of `vpc.enableVPCEndpoints` was changed to `false` to prevent Konvoy unexpectedly modifying the endpoints in user provided VPCs.
+    This value should already be present in your `cluster.yaml` file, below is a partial `cluster.yaml` that contains the value you may add to retain the previous behavior of deploying VPC endpoints in your cluster.
+    This resource should only be required where the networking configuration of the cluster does not allow for direct access to the AWS API.
+
+    ```yaml
+    kind: ClusterProvisioner
+    apiVersion: konvoy.mesosphere.io/v1beta1
+    metadata:
+      name: konvoy
+    spec:
+      provider: aws
+      aws:
+        vpc:
+          enableVPCEndpoints: true
+    ```
+
+#### Improvements
+
+- Update default Kubernetes version to `v1.16.8` which fixes a CVE found in a previous version.
+- Update Calico to `v3.13.1` which fixes a CVE found in a previous version.
+- Update Ansible to `2.7.16.0` which fixes a CVE found in a previous version.
```

Comment:
Is it worth mentioning which CVEs get resolved? Like the actual CVE number and link

Reply:
Added for the Kubernetes and Calico links, the Ansible bump one contains multiple lower level vulnerabilities.

Reply:
thanks I added a little bit more